equivalence.red

```diff
@@ -0,0 +1,14 @@
+import path
```

Comment:
can you also have `invariance.red` import `equivalence.red`?

Reply:
Done.